spec/02_find_length_strings_in_array_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Specification for RubyMonk Primer Problem 2 Find the length of strings in an array
+require_relative '../02_find_length_strings_in_array'
+
+RSpec.describe FindLengthStringsInArray do
+  subject(:strings_in_array) { described_class.new }
+
+  describe '#length_finder' do
+    # rubocop:disable Style/WordArray
+    it 'given ["I","am","genius"] it returns [1,2,6]' do
```

Comment:
would prefer a higher-level description for this method, too, as well as splitting out the `context` and `it` that are secretly hidden in this description.  Something like:
```
describe #length_finder do
  context 'when given an list of words' do
      it 'returns a list of the lengths of each word'
  end
end
```

Reply:
I agree but I was really trying to keep the original tests as best as I could reverse engineer them.  I definitely would not write tests this way.